docs/syzbot.md

```diff
@@ -164,8 +164,20 @@ so please treat the results with understanding. You may consult the provided
 Suggestions and patches that improve bisection quality for common cases are
 [welcome](https://github.com/google/syzkaller/issues/1051).
 
+Syzbot supports cause bisection(find the commit that introduces a bug) and fix bisection(find the commit that fixes a bug).
+
+The web UI for a specific kernel(say [upstream linux](https://syzkaller.appspot.com/upstream)) shows the `Bisected` status for all bugs.
+
 <div id="bisection"/>
 
+### Fix Bisection
+Syzkaller will perform fix bisection on bugs that meet the following criterion:
+- The kernel tree corresponding to the bug has fix bisection enabled(for instance, fix bisection might be disabled on kernel trees that have commits which are force pushed).
```

Comment:
We try to keep most lines in this this doc at 80 columns. Please split long lines.

Reply:
Thanks! I've updated the PR with the suggested changes above